src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -981,6 +1005,36 @@ public DSModelElementsSelection(IEnumerable<string> selectionIdentifier, IEnumer
         { }
     }
 
+    [NodeName("Pick Model Elements")]
```

Comment:
Or perhaps "Select Model Elements with the Pick Editor"

Reply:
Yes we will keep the shorter for now